runtime/interface.go

```diff
@@ -69,10 +52,16 @@ type Interface interface {
 	SetValue(owner, key, value []byte) (err error)
 	// CreateAccount creates a new account.
 	CreateAccount(payer Address) (address Address, err error)
+	// AddEncodedAccountKey appends an encoded key to an account.
+	AddEncodedAccountKey(address Address, publicKey []byte) error
+	// RemoveEncodedAccountKey removes a key from an account by index, add returns the encoded key.
+	RemoveEncodedAccountKey(address Address, index int) (publicKey []byte, err error)
```

Comment:
Given that the key is not removed, but revoked, maybe rename this to `RevokeEncodedAccountKey` (The Cadence function is also named `revoke`). It should have been named "revoke" in the first place.